src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/ApplicationDeployerFactory.cs

```diff
@@ -18,7 +18,7 @@ public class IISApplicationDeployerFactory
         /// <param name="deploymentParameters"></param>
         /// <param name="loggerFactory"></param>
         /// <returns></returns>
-        public static ApplicationDeployer Create(DeploymentParameters deploymentParameters, ILoggerFactory loggerFactory)
+        public static ApplicationDeployer Create(IISDeploymentParameters deploymentParameters, ILoggerFactory loggerFactory)
```

Comment:
This is a non-starter, it breaks every other repo.

Reply:
Agreed. Thoughts on how we are going to have ServerTests use the IISApplicationDeployerFactory?

Reply:
It already does. Just don't force argument to be `IISDeploymentParameters `